### PR TITLE
updated split and merge rules

### PR DESCRIPTION
**Pull request by gmafrafortuna**

### CODE REVIEW

File: Snakemake/envs/shapeit.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+channels:
+  - conda-forge
+  - bioconda
+  - main
+dependencies:
+  - shapeit4 = 4.2.2
```
Comment (janaobsteter) [inline Snakemake/envs/shapeit.yaml:6]: Gabriela, can you remove these files? I think this is now all in workflow/

File: Snakemake/workflow/rules/merge_samples_vcf.smk [hunk 2]
```diff
@@ -1,71 +1,99 @@
-# ---------------------------------------------------------------------------- #
-#                               MERGE                                          #
-# ---------------------------------------------------------------------------- #
+import yaml
+vcfdir = "vcfDir"
+chromosomes = [f'Chr{n}' for n in range(1, 3)]
```
Comment (janaobsteter) [inline Snakemake/workflow/rules/merge_samples_vcf.smk:3]: Does this stay here or goes in the Snakemake (I know this is next step)

File: Snakemake/workflow/rules/merge_samples_vcf.smk [hunk 3]
```diff
@@ -1,71 +1,99 @@
...
-    input: [f'{vcfdir}/{{chromosome}}/{{file}}.txt'
-    output: temp([f'{vcfdir}/{{chromosome}}/{{file}}.filtered')
-    log: expand('logs/{{chromosome}}_{file}.log', file=allfiles)
-    run:
-        from itertools import combinations
-        samplelist=[]
-        for ifile in input:
-            with open(ifile, 'r') as f:
-                samplelist.append(f.read().splitlines())
+rule all:
```
Comment (janaobsteter) [inline Snakemake/workflow/rules/merge_samples_vcf.smk:13]: WE probably also remove this in the next step, right? When we edit the Snakefile

File: Snakemake/envs/shapeit.yaml (hunk 1 above)
Comment (gmafrafortuna) [reply Snakemake/envs/shapeit.yaml]: removed all of them

File: Snakemake/workflow/rules/merge_samples_vcf.smk (hunk 2 above)
Comment (gmafrafortuna) [reply Snakemake/workflow/rules/merge_samples_vcf.smk]: this is for the snakefile  (I forgot to remove. done now). not sure about the import yaml so I kept here

File: Snakemake/workflow/rules/merge_samples_vcf.smk (hunk 3 above)
Comment (gmafrafortuna) [reply Snakemake/workflow/rules/merge_samples_vcf.smk]: removed all rule alls

**Review by janaobsteter — APPROVED** [review]

**Pull request merged**